src/pages/newAquarium/index.tsx

```diff
@@ -0,0 +1,123 @@
+import { Col, Container, Row } from 'react-bootstrap';
+import Form from 'react-bootstrap/Form';
+import { useTranslation } from 'react-i18next';
+
+import { Button, Card, Input, Tile } from '@/components';
+
+export const NewAquariumView: React.FC = () => {
+  const { t } = useTranslation();
+  return (
+    <Form>
+      <h3 className="mb-4">{t('aquarium_setup')}</h3>
+      <Row>
+        <Col md>
+          <Tile title={t('parameters')}>
+            <Container className="px-3">
+              <Row>
+                <Col>
+                  <Form.Group controlId="aquariumName">
+                    <Form.Label className="mt-2">
+                      <h4>{t('aquarium_name')}</h4>
+                    </Form.Label>
+                    <Input placeholder={t('name')} />
+                  </Form.Group>
+                </Col>
+              </Row>
+              <Row>
+                <Col>
+                  <Form.Group controlId="temperature">
+                    <Form.Label className="mt-3">
+                      <h4>{t('water_temperature')}</h4>
+                    </Form.Label>
+                    <Input placeholder={t('temperature')} />
+                  </Form.Group>
+                </Col>
+                <Col>
+                  <Form.Group controlId="targetPh">
+                    <Form.Label className="mt-3">
+                      <h4>{t('target_water_ph')}</h4>
+                    </Form.Label>
+                    <Input placeholder={t('water_ph')} />
+                  </Form.Group>
+                </Col>
+              </Row>
+              <Form.Label className="mt-3">
+                <h4>{t('lights_on_time_interval')}</h4>
+              </Form.Label>
+              <Row>
+                <Col>
+                  <Form.Group controlId="lightStartTime">
+                    <Input placeholder={t('hour_from')} />
+                  </Form.Group>
+                </Col>
+                <Col>
+                  <Form.Group controlId="lightStopTime">
+                    <Input placeholder={t('hour_to')} />
+                  </Form.Group>
+                </Col>
+              </Row>
+              <Form.Group controlId="frequency">
+                <Form.Label className="mt-3">
+                  <h4>{t('measurements_frequency')}</h4>
+                </Form.Label>
+                <Row>
+                  <Col sm>
```

Comment:
Tutaj też `sm` nie ma wartości, coś powinno być tutaj wpisane

Reply:
W ten sposób ustawiam wartość `sm` na `true` (tak jak sie robi np. z `disabled` na buttonach), jest to przewidziane w dokumentacji do robienia breakpointa dzielącego na równe kolumny
![image](https://user-images.githubusercontent.com/73584389/234945762-ffec6d09-4cdd-418d-bad0-1e691eaa3b4e.png)